src/mgr/BaseMgrModule.cc

```diff
@@ -1064,6 +1070,9 @@ PyMethodDef BaseMgrModule_methods[] = {
   {"_ceph_get_version", (PyCFunction)ceph_get_version, METH_VARARGS,
    "Get the ceph version of this process"},
 
+  {"_ceph_get_release_name", (PyCFunction)ceph_get_release_name, METH_VARARGS,
```

Comment:
the calling convention of this method should be `METH_NOARGS`. as it accept no argument than `self`. see https://docs.python.org/3/c-api/structures.html

Reply:
Oh you're right. I copied the code from the ``ceph_get_version`` function. I will fix it there, too.